<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customDetails</class>
 <widget class="QMainWindow" name="customDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblHeading">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Custom Details</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblApiKey">
            <property name="text">
             <string>API Key : </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leApiKey"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblApiSecretKey">
            <property name="text">
             <string>API Secret Key : </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbProfitThrld">
            <property name="text">
             <string>Profit Threshold : </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblAverageThrld">
            <property name="text">
             <string>Average Threshold : </string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblQuantity">
            <property name="text">
             <string>Quantity : </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leApiSecretKey"/>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="dsbProfitThrld"/>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="dsbAverageThrld"/>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="sbQuantity"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lblDesktopNoti">
            <property name="text">
             <string>Desktop Notification : </string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lblWhatsappNoti">
            <property name="text">
             <string>Whatsapp Notification : </string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lblWhatsappNo">
            <property name="text">
             <string>Whatsapp No : </string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="leWhatsappNo"/>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="cbDesktopNoti">
            <property name="text">
             <string>CheckBox</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="cbWhatsappNoti">
            <property name="text">
             <string>CheckBox</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
